cfstart.sh

```diff
@@ -36,6 +37,9 @@ REDIS_PASSWORD=$(echo $VCAP_SERVICES | jq -r --arg SVC_REDIS $SVC_REDIS '.[][] |
 REDIS_PORT=$(echo $VCAP_SERVICES | jq -r --arg SVC_REDIS $SVC_REDIS '.[][] | select(.name == $SVC_REDIS) | .credentials.port')
 export CKAN_REDIS_URL=rediss://:$REDIS_PASSWORD@$REDIS_HOST:$REDIS_PORT
 
+# We need the secret credentials for various application components (DB configuration, license keys, etc)
+export CKAN___BEAKER_SESSION_SECRET=$(echo $VCAP_SERVICES | jq -r --arg SVC_SECRETS $SVC_SECRETS '.[][] | select(.name == $SVC_SECRETS) | .credentials.BEAKER__SESSION__SECRET')
```

Comment:
In the credentials, I would just name this the same as the env variable `CKAN___BEAKER_SESSION_SECRET`. More descriptive and fewer layers of indirection.